docs/integrations/index.md

```diff
@@ -0,0 +1,18 @@
+---
+description: |
+  Various projects can integrate into Fresh to easily add functionality to your
+  project.
+---
+
+This section of the docs showcases modules or other projects that integrate with
+Fresh. These integrations can be used to add functionality to your project.
+
+## `fresh_charts`
+
+Fresh charts allows for easy integration of [Chart.js][chart-js] charts into
```

Comment:
missing the named link?

Reply:
addressed in fcc3a306f7f9994c341a24e57fca00ad1c3ad87d